wp-content/themes/core/pcss/base/typography/_lists.pcss

```diff
@@ -48,36 +51,56 @@ li {
 .list-number,
 .t-content ol {
 	list-style: decimal outside;
-	padding-left: 1.5em;
+	padding-left: 0;
 	padding-right: 0;
 
 	@media (--viewport-medium) {
-		padding-left: 3em;
 		padding-right: 3em;
 	}
-}
 
-/* -------------------------------------------------------------------------
- * DL
- * ------------------------------------------------------------------------- */
+	li {
+		counter-increment: item;
+		font-size: var(--font-size-body-s);
+		list-style-type: none;
+		position: relative;
 
-.dl,
-.t-content dl {
-	padding-left: 1.5em;
-	padding-right: 0;
+		@media (--viewport-medium) {
+			font-size: var(--font-size-body);
+		}
 
-	@media (--viewport-medium) {
-		padding-left: 3em;
-		padding-right: 3em;
+		&:before {
+			content: counter(item);
+			font-family: var(--font-family-calibre);
+			font-weight: var(--font-weight-semibold);
+			line-height: var(--line-height-base);
+			float: left;
```

Comment:
Curious, how robust is using float for this? Does it hold up for wrapping list item text as well? I've typically done these as a position absolute implementation, not wrong, just different.

Reply:
The float allowed wrapping. Refactored w/ absolute positioning.